packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -461,12 +461,12 @@ export interface IPerfSignalReport {
 	roundTripSignalSequenceNumber: number | undefined;
 
 	/**
-	 * Next expected signal sequence number to be received.
+	 * Next expected signal sequence number to be received. Upper bound of signal monitoring window.
 	 */
 	trackingSignalSequenceNumber: number | undefined;
 
 	/**
-	 * Minimum expected signal sequence number to be received.
+	 * Lower bound of signal monitoring window.
 	 */
 	minimumTrackingSignalSequenceNumber: number | undefined;
```

Comment:
Is this inclusive or exclusive? It looks exclusive currently based on the one bump to a received signal. (In my experience "minimum"s are rarely exclusive. People think "if minimum is 2, then 2 is a valid consideration.") (I did not look at all other uses of the value.)

Reply:
It is technically inclusive -- if a client reconnects and sends signal 134, minimumtrackingSignalSequenceNumber will be 134 and will include 134 within the monitoring window when processed. Every time a SignalLatency event is logged, the minimumTrackingSignalSeqNumber is reset to the signal that triggered the event (the latency tracked signal). So, this means that the value always set to a signal that is has been or is being tracked, so it is technically inclusive.

Reply:
Okay. Always inclusive. Sometimes it tracks something we haven't yet received back and other times it tracks something we have. That may make it a little hard to use.
If we were to assume that when it tracks something we have received we'd never receive that again, then inclusive makes plenty of sense. Well, I'll need to go read the logic....
In the meantime, an important consideration is that we can receive the same message twice. This should be pretty rare, but it will happen - networks are not perfect. So, we should know what happens when we get something twice. I do not think we want perfect tracking of everything; so, getting a duplicate of not the latest _could_ look like an out of order signal.
We can surely simulate duplicate signal sends with our current tooling. Instead of trying to cover those now, we can add comments. Better duplicate signal handling can be a follow on.